### ISSUE TITLE
Material UI entry border disappears on certain Android OS

### ISSUE THREAD

**Issue by Gevand** [xamarin/Xamarin.Forms#13642]
Xamarin.Forms = 5.0.0.1931
Xamarin.Forms.Visual.Material = 5.0.0.1931
uses-sdk android:minSdkVersion="21" 
android:targetSdkVersion="29" 
build set up to link sdk only

After updating Xamarin.Forms and Xamarin.Forms.Visua.Material packages, its seems that on newer Android OS versions the lines on entry boxes, editors, date pickers, etc disappear. They will only reappear if the textbox is focused.

Android Version: 10
Kernel Version: 4.4.210-perf+
Build Number: QQ1A.200105.032

![image](https://user-images.githubusercontent.com/2449707/106808859-ce887200-661f-11eb-9ab6-c238b7c0139b.png)

This is not the case for Older Android, always works fine in IOS. 

Android Version: 10
Kernel Version: 3.18.137-g72a7a64494e
Build Number: QP1A101005.007.A3

![image](https://user-images.githubusercontent.com/2449707/106809168-3474f980-6620-11eb-8606-c855bf9879f3.png)

**Comment by rachelkang** [comment]
Hi, @Gevand - thanks for submitting this issue. Unfortunately, I wasn't able to reproduce the behavior you're describing, and I'm still able to see all the underlines - would you share your project or a sample project that reproduces this behavior so that we can take a closer look? Thanks :)

**Comment by Gevand** [comment]
@rachelkang 

Thank you for a quick response. I did some more digging and OS version has nothing to do with the issue. All the phones that had problems had dark mode on.

What I discovered is if you take this style:

```xml
<Style x:Key="normalEntryStyle" TargetType="{x:Type Entry}">
            <Setter Property="PlaceholderColor" Value="Blue"/>
            <Setter Property="BackgroundColor" Value="White"></Setter>
            <Setter Property="Visual" Value="Material"></Setter>
</Style>
```

And apply it to an entry, the line at the bottom should become the same color as the placeholder (blue).

```xml
 <Entry WidthRequest="300" Placeholder="Some Text" Style="{StaticResource normalEntryStyle}" ></Entry>
```

This was the behavior in version 4.7.0.968. The bottom line is no longer the placeholder color unless focused on. When dark mode is turned on the line become white (in previous version it would stay blue), and since the background color of the textbox is white, its not possible to see it.
